tests/unit/records/test_jobs_hints_and_records_format.py

```diff
@@ -2,6 +2,7 @@
 from odictliteral import odict
 from mock import Mock, patch
 from records_mover.records.job.mover import run_records_mover_job
+from records_mover.records.records_format import ParquetRecordsFormat
```

Comment:
This file, one of the 3 largest in the project, increased in size to 362 lines.  The total size of those files is now 3404 lines (target: 1087).  Is this file complex enough to refactor?